

kernel/power/smp.c: In function `smp_pause':
kernel/power/smp.c:25: warning: implicit declaration of function `__save_processor_state'
kernel/power/smp.c:42: warning: implicit declaration of function `__restore_processor_state'



---

 25-akpm/include/asm-i386/suspend.h |    2 ++
 1 files changed, 2 insertions(+)

diff -puN include/asm-i386/suspend.h~next-step-of-smp-support-fix-device-suspending-warning-fix include/asm-i386/suspend.h
--- 25/include/asm-i386/suspend.h~next-step-of-smp-support-fix-device-suspending-warning-fix	2004-06-27 23:48:22.961763552 -0700
+++ 25-akpm/include/asm-i386/suspend.h	2004-06-27 23:48:22.966762792 -0700
@@ -38,6 +38,8 @@ struct saved_context {
 
 extern void save_processor_state(void);
 extern void restore_processor_state(void);
+void __save_processor_state(struct saved_context *ctxt);
+void __restore_processor_state(struct saved_context *ctxt);
 
 #ifdef CONFIG_ACPI_SLEEP
 extern unsigned long saved_eip;

_
